tutorials/02a_RunningREMINDLocally.md

```diff
@@ -0,0 +1,37 @@
+# Running REMIND locally
+Mika Pflüger (mika.pflueger@pik-potsdam.de)
+
+*Note:* This tutorial is not yet finished, not all steps needed are described, yet.
+
+## Download Input Data Automatically
+
+If you have access to the PIK cluster and like to automatically download the input data when running REMIND on your local machine, you need to configure this via environment variables.
```

Comment:
maybe we should add here a comment that only PIK members have access to the PIK cluster. Otherwise, even more external people ask for access to the PIK cluster